_posts/en/2019-01-20-rbf-in-the-wild.md

```diff
@@ -0,0 +1,242 @@
+---
+title: 'RBF in the Wild'
+permalink: /en/rbf-in-the-wild/
+name: 2019-01-20-rbf-in-the-wild
+type: posts
+layout: post
+lang: en
+version: 1
+---
+
+Opt-in Replace by Fee was standardized in December 2015. But who is supporting it and what is the user experience like? This post is a study of Bitcoin wallet and block explorer support and usability of Opt-in RBF (BIP125). The findings were initially presented during the most recent [Bitcoin Optech Workshop in Paris](https://bitcoinops.org/workshops/).
```

Comment:
Minor suggestions:

- Put "(RBF)" after the first introduction of this term so any Bitcoin users who have been hiding under rocks for the last three years will know what the subsequent uses of the abbreviation refer to.

- You may also want to link to BIP125; you can do that by simply changing "(BIP125)" to "([BIP125][])" as the link is defined in the references file you already include.

- Change "https://bitcoinops.org/workshops/" to "/workshops/" (a full-qualified link to a relative link) as Travis CI will automatically test internal relative links to ensure they're correct (which helps a lot if we move something and break old links).